.github/workflows/py-coding-style.yml

```diff
@@ -0,0 +1,48 @@
+name: Python coding style
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+
+env:
+  DEBUG: 1
+
+jobs:
+  py-coding-style:
+    name: Python coding style
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Cache pip packages
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-pip-modules
+        with:
+          path: ~/.pip-cache
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/requirements.txt') }}-${{ hashFiles('**/requirements-dev.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - name: Install dependencies
+        run: pip install -r backend/requirements.txt -r backend/requirements-dev.txt
+
+      - name: Formatting
+        run: black --check backend/
+
+      - name: Linting
+        run: flake8 backend/
```

Comment:
Should probably be something like `cd backend && flake8`, since that's the root of the Python project here.